snippets/shuffle.md

```diff
@@ -1,10 +1,20 @@
 ### shuffle
 
-Randomizes the order of the values of an array.
+Randomizes the order of the values of an array, in place.
 
-Use `Array.sort()` to reorder elements, using `Math.random()` in the comparator.
+Users the Fisher-Yates reorder the elements, based on the [Lodash implementation](https://github.com/lodash/lodash/blob/b2ea6b1cd251796dcb5f9700c4911a7b6223920b/shuffle.js).
```

Comment:
Users?
Is it supposed to be `Uses the Fisher-Yates algoritm to reorder the elements of the array, based on the [Lodash implimentation](https://github.com/lodash/lodash/blob/b2ea6b1cd251796dcb5f9700c4911a7b6223920b/shuffle.js)`